File: PresentationLayer/Sep3Blazor-Soap/Sep3Blazor/Model/Note.cs
using System;

namespace Sep3Blazor.Model
{
    public class Note 

    {

        public int id{ get; set; }

        public int user_id { get; set; }

        public int group_id { get; set; }
    
        public int week{ get; set; }
 
        public int year{ get; set; }
  
        public String name{ get; set; }

        public String status{ get; set; }

        public String text{ get; set; }

        public Note(int id, int userId, int groupId, int week, int year, string name, string status, string text)
        {
            this.id = id;
            this.user_id = userId;
            this.group_id = groupId;
            this.week = week;
            this.year = year;
            this.name = name;
            this.status = status;
            this.text = text;
        }

        // public Note(int week, int year, string name, string status, string text)
        // {
        //     this.week = week;
        //     this.year = year;
        //     this.name = name;
        //     this.status = status;
        //     this.text = text;
        // }
    }
}

File: PersistenceServer/Sep3JavaServer-master/src/main/java/PersistenceServer/Note.java
package PersistenceServer;

public class Note {

   private int id,user_id, group_id,week,year;
   private  String name, status, text;

    public Note(int id, int user_id, int group_id, int week, int year, String name, String status, String text) {
        this.id = id;
        this.user_id = user_id;
        this.group_id = group_id;
        this.week = week;
        this.year = year;
        this.name = name;
        this.status = status;
        this.text = text;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {this.id = id;}

    public int getUserId() {return user_id;}

    public void setUserId(int userId) {
        this.id = id;
    }

    public int getGroupId() {
        return group_id;
    }

    public void setGroupId(int groupId) {
        this.id = id;
    }

    public int getWeek() {
        return week;
    }

    public void setWeek(int week) {
        this.week = week;
    }

    public int getYear() {
        return year;
    }

    public void setYear(int year) {
        this.year = year;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }
}
